tests/phpunit/tests/KSES.php

```diff
@@ -308,6 +332,25 @@ public function test_filter_kses_allowed_html( $html, $expected ) {
 		remove_filter( 'wp_kses_allowed_html', [ $kses, 'filter_kses_allowed_html' ] );
 	}
 
+	/**
+	 * Testing the filter_kses_allowed_html() method.
+	 *
+	 * @covers ::filter_kses_allowed_html
+	 * @covers ::array_merge_recursive_distinct
+	 */
+	public function test_filter_kses_allowed_html_filter() {
+		$kses         = new \Google\Web_Stories\KSES();
+		$allowed_tags = [
+			'img' => [
+				'width' => true,
```

Comment:
Let's actually add `width` to the list of `img `attributes in the KSES class. That's the P0 concern.

And then use some other `$allowed_tags` value for this test of ccourse.

Reply:
Done.